zerver/openapi/zulip.yaml

```diff
@@ -7057,30 +7074,35 @@ paths:
                             "total_messages_in_topic": 5,
                           }
                         description: |
-                          A special failed JSON response for when the user has permission to move
+                          A special failed JSON response (`"code": "MOVE_MESSAGES_TIME_LIMIT_EXCEEDED"`)
```

Comment:
Good idea. The way we render these example responses gets pretty confusing once their descriptions are more than a line or two long:
![image](https://github.com/zulip/zulip/assets/28173/3d6e9f7c-d464-4eaa-bc68-184f4f32ba36)

in that it looks very much like the paragraph "A special failed JSON response …" describes the example immediately *above* it, rather than the example far below. (When they're all just a line long, the colon at the end of the line does an OK job at disambiguating.)

Ideally we'd see about fixing that at some point. But in the meantime, saying `"code": "MOVE_MESSAGES_TIME_LIMIT_EXCEEDED"`` here should do a lot to help people determine quickly that it's not about the example above, so they can look again and puzzle out that it's about the example far below.